addon/components/denali-multi-select.hbs

```diff
@@ -0,0 +1,35 @@
+{{!-- Copyright 2020, Verizon Media. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<BasicDropdown @tagName="div" @matchTriggerWidth={{true}} @verticalPosition="below" @renderInPlace={{true}} class="multi-select {{this.isSmallClass}} {{this.isInverseClass}}" as |dd|>
+  <ul  {{on "click" dd.actions.toggle}} class="selections has-arrow {{if dd.isOpen "activate" ""}}" data-ebd-id="{{dd.uniqueId}}-trigger" ...attributes>
+    {{#if (eq this.selections.length 0)}}
+      <li class="select">{{placeholder}}</li>
```

Comment:
Is this using a denali class/style or can we bem-ify?

Reply:
I tried to name classes similarly to how denali ones are named but we can bem-ify for sure